cauldron/redis_cache.py

```diff
@@ -270,4 +278,240 @@ def apply_cache(*args, **kwargs):
 
             return apply_cache
 
+        return wrapped
+
+class RedisCacheV2:
```

Comment:
we have implemented  RedisCacheV2 for what different purpose, are we using new redis version?